### PR TITLE
Imb 155 Update Accessibility Statement

### PR DESCRIPTION
**Pull request by muthrajan**
## What?
[IMB-155 Update Accessibility Statement](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-155)
 
## Why?
This accessibility statement applies to the IMA website and navigate most of the website using just a keyboard, navigate most of the website using speech recognition software and listen to most of the website using a screen reader 
 
## How?
- In common folder accessibility with header was added to pages.json
- In views/content accessibility.md was added with all the necessary text
- In views accessibility.html was added
- In common index.js was added
- Server.spec.js was amended with stub
- PR changes
 
## Testing?
Tested locally

### CODE REVIEW

File: apps/common/translations/src/en/buttons.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "start-home": "Start Now"
+}
```
Comment (jamiecarterHO) [inline apps/common/translations/src/en/buttons.json:3]: Is this button in the accessibility page? If not I don't think we need it

File: apps/common/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  name: 'common',
+  steps: {
+    '/start': {
+      template: 'start',
+      next: 'ima/continue-form'
+    },
```
Comment (jamiecarterHO) [inline apps/common/index.js:9]: I don't think we need the start route in the common app as it comes from the IMA app

File: apps/common/fields/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (jamiecarterHO) [inline apps/common/fields/index.js:1]: Does the app still build if we remove this file?

File: apps/common/translations/src/en/pages.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "start": {
+    "header": "Illegal migration act"
+  },
+  "declaration": {
+    "header": "Declaration"
+  },
```
Comment (jamiecarterHO) [inline apps/common/translations/src/en/pages.json:7]: Think we can remove these from common - they are already in the IMA app.

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]
Some questions about code we won't use in the common app

File: apps/common/translations/src/en/buttons.json (hunk 1 above)
Comment (muthrajan) [reply apps/common/translations/src/en/buttons.json]: Removed

File: apps/common/index.js (hunk 2 above)
Comment (muthrajan) [reply apps/common/index.js]: Start route removed

File: apps/common/fields/index.js (hunk 3 above)
Comment (muthrajan) [reply apps/common/fields/index.js:1]: It is necessary as it is not building

File: apps/common/translations/src/en/pages.json (hunk 4 above)
Comment (muthrajan) [reply apps/common/translations/src/en/pages.json]: Removed

File: apps/common/index.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  name: 'common',
+  steps: {
+    '/start': {
+      next: 'ima/continue-form'
+    },
```
Comment (jamiecarterHO) [inline apps/common/index.js:8]: Sorry missed this one, can we do without the start step or is it needed to build?

**Review by jamiecarterHO — COMMENTED** [review]
One more question.

File: apps/common/index.js (hunk 5 above)
Comment (muthrajan) [reply apps/common/index.js]: Removed

**Review by jamiecarterHO — APPROVED** [review]
There is a new vulnerability to whitelist in the snyk scan, but the accessibility code looks good.

**Pull request merged**